modules/backend/widgets/Lists.php

```diff
@@ -1077,7 +1077,7 @@ public function getColumnValue($record, $column)
          * Apply default value.
          */
         if ($value === '' || $value === null) {
```

Comment:
Change the conditional to `if (($value === '' || is_null($value)) && !empty($column->defaults)) {`

Reply:
@LukeTowers done